src/Mobs/Monster.h

```diff
@@ -74,6 +76,32 @@ class cMonster :
 	virtual void CheckEventSeePlayer(cChunk & a_Chunk);
 	virtual void EventSeePlayer(cPlayer * a_Player, cChunk & a_Chunk);
 
+	// tolua_begin
```

Comment:
Add an empty line after this, to make it stand out more. Same with the finalizing `// tolua_end`